client/src/main/java/org/eclipse/hono/client/impl/AbstractHonoClient.java

```diff
@@ -221,4 +223,49 @@ protected static final void setApplicationProperties(final Message msg, final Ma
             msg.setApplicationProperties(applicationProperties);
         }
     }
+
+    /**
+     * TODO.
+     */
+    protected final void startAutoCloseTimer() {
+        final long inactiveLinkTimeout = connection.getConfig().getInactiveLinkTimeout();
+        if (inactiveLinkTimeout > 0) {
+            startAutoCloseTimer(inactiveLinkTimeout);
+        }
+    }
+
+    /**
+     * Stores the current time stamp on the sender link. This is used to detect and close inactive AMQP sender links.
+     */
+    protected final void storeLastSendTime() {
+        sender.attachments().set(ATTACHMENT_LAST_SEND_TIME, Long.class, Instant.now().toEpochMilli());
```

Comment:
I am not sure but I do not think that we need to create an `Instant` object here but can simply use good old `System.currentTimeMillis()`. WDYT?